spanner/ddl/ast.go

```diff
@@ -195,14 +195,23 @@ func (cd ColumnDef) PrintColumnDef(c Config) (string, string) {
 //	key_part:
 //	   column_name [{ ASC | DESC }]
 type IndexKey struct {
-	Col   string
+	ColId string
 	Desc  bool // Default order is ascending i.e. Desc = false.
 	Order int
 }
 
 // PrintIndexKey unparses the index keys.
-func (pk IndexKey) PrintIndexKey(c Config) string {
-	col := c.quote(pk.Col)
+func (idx IndexKey) PrintIndexKey(ct CreateTable, c Config) string {
+	col := c.quote(ct.ColDefs[idx.ColId].Name)
+	if idx.Desc {
+		return fmt.Sprintf("%s DESC", col)
+	}
+	// Don't print out ASC -- that's the default.
+	return col
+}
+
+func (pk IndexKey) PrintPkKey(ct CreateTable, c Config) string {
```

Comment:
we can use a single method instead of 2

Reply:
Updated to used PrintPkOrIndexKey common method. Please review it.